walkers/src/tiles.rs

```diff
@@ -94,15 +100,17 @@ impl Tiles {
             tile_size,
         }
     }
+}
 
+impl TilesManager for Tiles {
```

Comment:
I really hate naming things `*Manager`, and this will probably change in the near future, once I figure what I like best. :) Perhaps I'll go for `trait Tiles` / `HttpTiles`, or maybe even merge it with `TileSource`.

Anyway, this is just to give a heads up, no need to change it now.

Reply:
> I really hate naming things *Manager

This name really bad. First I want to name as 'TIle*s*Source', but 'TileSource' already exists and I was afraid of confusion

Reply:
Additionally, I think 'cache' in `Tiles` has not correct name, or not correctly placed:

atually, it is a display cache, e.g. to display already prepared textures, not to get tile in fast way.
it would be good to limit this cache to the size of the viewport to avoid uncontrolled memory consumption